qemu/tests/x86_cpu_test_0x40000001.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from virttest import error_context
+from virttest import data_dir
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check kvm 0x40000001 inside guest.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+    nums_cpu = str(vm.cpuinfo.smp)
+
+    test_dir = params["test_dir"]
+    source_file = params["source_file"]
+    src_cpuid = os.path.join(data_dir.get_deps_dir(), source_file)
+    vm.copy_files_to(src_cpuid, test_dir)
+    try:
+        session.cmd('tar -xzf /home/cpuid.src.tar.gz -C /home/ && cd /home/cpuid-20201006 && make')
```

Comment:
Here reference `test_dir` and `source_file`, I think that's why we define them.

Reply:
Thanks and update.